.github/std-unexploded.yml

```
@@ -51,6 +51,9 @@ jobs:
       #   with:
       #     nixbuild_ssh_key: ${{ secrets.SSH_PRIVATE_KEY }}
       #     generate_summary_for: job
+
+      - uses: ./.github/workflows/show-commit.yaml
```

```suggestion
      - uses: ./.github/actions/show-commit.yaml
```
The convention is to have a `actions` dir